requests_pkcs12.py

```diff
@@ -37,7 +37,7 @@ def check_cert_not_after(cert):
     if cert_not_after < datetime.datetime.utcnow():
         raise ValueError('Client certificate expired: Not After: {cert_not_after:%Y-%m-%d %H:%M:%SZ}'.format(**locals()))
 
-def create_sslcontext(pkcs12_data, pkcs12_password_bytes, ssl_protocol=default_ssl_protocol):
+def create_sslcontext(pkcs12_data, pkcs12_password_bytes=None, ssl_protocol=default_ssl_protocol):
```

Comment:
Given that `pkcs12_password_bytes=None` is a pretty unsafe default, it's better to not have any default value for `pkcs12_password_bytes`.